fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/db2/Db2CreateTablespace.java

```diff
@@ -24,12 +24,10 @@
     private final String tablespaceName;
     private final int extentSizeKB;
 
-
-    /**
-     * Public constructor
-     * @param tableName
-     * @param partitionId
-     */
+/**
+ * 
+ * @param tablespaceName
+ */
```

Comment:
indentation seems off

Reply:
interesting... not sure when that was introduced